tasks/main.yml

```diff
@@ -236,14 +236,18 @@
           - "mssql_password"
           - "mssql_edition"
 
+    # Setting MSSQL_SA_PASSWORD inline for security because setting it with
+    # environment: reveals the value when running playbooks with high verbosity
     - name: Set up MSSQL
-      command: "{{ __mssql_conf_cli }} -n setup"
+      shell: >-
+        MSSQL_SA_PASSWORD={{ mssql_password | quote }}
+        {{ __mssql_conf_cli }} setup
```

Comment:
This does not show the shell command in the output, even with high verbosity?

Reply:
Yes, the whole task is hidden.